.rubocop.yml

```diff
@@ -186,6 +191,7 @@ Style/FormatStringToken:
 # https://docs.rubocop.org/rubocop/cops_style.html#stylehashsyntax
 Style/HashSyntax:
   EnforcedStyle: ruby19_no_mixed_keys
+  EnforcedShorthandSyntax: either
```

Comment:
Explanation in the commit - https://github.com/mastodon/mastodon/pull/29702/commits/6b4f763cb5434e1938863e17ae7776e27a8489ad